### PR TITLE
Additional fixes plus updates to the build matrix and pypi classifiers, 3rd and final attempt

### PR DESCRIPTION
**Pull request by webtweakers**

### CODE REVIEW

**Comment by webtweakers** [comment]
Ok, build passes. Sorry for the mess...

File: tinymce/widgets.py [hunk 1]
```diff
@@ -42,7 +42,8 @@ def get_language_config():
     if mce_settings.USE_SPELLCHECKER:
         from enchant import list_languages
         enchant_languages = list_languages()
-        logger.info('Enchant languages: {0}'.format(enchant_languages))
+        if settings.DEBUG:
+            logger.info('Enchant languages: {0}'.format(enchant_languages))
```
Comment (browniebroke) [inline tinymce/widgets.py]: Maybe drop the if statement, and call `logger.debug` instead?

File: tinymce/widgets.py [hunk 2]
```diff
@@ -144,7 +145,11 @@ def media(self):
         js = [mce_settings.JS_URL]
         if mce_settings.USE_FILEBROWSER:
             js.append(reverse('tinymce-filebrowser'))
-        css = {'all': [mce_settings.CSS_URL or reverse('tinymce-css')]}
+        if mce_settings.ADDITIONAL_JS_URLS:
+            js.append(mce_settings.ADDITIONAL_JS_URLS)
```
Comment (browniebroke) [inline tinymce/widgets.py]: The doc says this setting is a list, so if you append to another list, you would have a nested list, wouldn't you?

**Comment by browniebroke** [comment]
Not sure if you are aware, but there is no need to close your pull requests and reopen a new one, you can keep pushing to your branch and it will update the existing pull request.

**Comment by webtweakers** [comment]
Yes, you're right on all accounts. :) Here's a fix.

**Comment by browniebroke** [comment]
Awesome work, thanks! I'll make a new release shortly.

**Comment by webtweakers** [comment]
Thanks, looking forward to it. Enjoy your vacation!

**Comment by browniebroke** [comment]
Released in [2.1.1](https://pypi.python.org/pypi/django-tinymce4-widget/2.1.1).

**Comment by webtweakers** [comment]
Already found it, installed it, deployed it. Works like a charm. Great work mate!

**Pull request merged**